WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.m

```diff
@@ -387,6 +387,11 @@ - (NSString *)postTypeToSync
     return PostServiceTypeAny;
 }
 
+- (NSDate *)lastSyncDate
```

Comment:
Since we're overriding in the subclasses this should probably be declared in the AbsractPostsListViewControllerSubclasses header.